sdk/identity/identity/src/credentials/workloadIdentityCredential.ts

```diff
@@ -123,4 +159,55 @@ export class WorkloadIdentityCredential implements TokenCredential {
     }
     return this.azureFederatedTokenFileContent;
   }
+
+  private async requestOidcToken(
+    oidcRequestUrl: string,
+    systemAccessToken: string,
+  ): Promise<string> {
+    console.log("Requesting OIDC token from Azure DevOps...");
+    console.debug(oidcRequestUrl);
+
+    const requestOptions = {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${systemAccessToken}`,
+      },
+    };
+
+    const response = await fetch(oidcRequestUrl, requestOptions);
```

Comment:
We should, ideally, use the core-rest-pipeline primitives unless there's a reason to use fetch directly